lib/rdoc/options.rb

```diff
@@ -325,11 +325,6 @@ class RDoc::Options
 
   attr_accessor :verbosity
 
-  ##
-  # URL of web cvs frontend
-
-  attr_accessor :webcvs
```

Comment:
To be on super super safe side, would it make sense to make this no-op? Seems there are still some old gems using this like https://github.com/sunspot/sunspot/blob/c9c17d3d0e5869d157d682059b864d758ca12a8b/sunspot/sunspot.gemspec#L35.

related GH search https://github.com/search?q=rdoc_options+webcvs+language%3ARuby+&type=code

Reply:
Or marking it as deprecated?

Reply:
The people who will see the deprecation probably aren't the author and can't fix it since it's in the gemspec.